cli/src/commands/schema/edit.ts

```diff
@@ -623,3 +678,44 @@ Beware that this can lead to ${chalk.bold(
     this.success('Migration completed!');
   }
 }
+
+function parseBoolean(value?: string) {
+  if (!value) return undefined;
+  const val = value.toLowerCase();
+  if (['true', 't', '1', 'y', 'yes'].includes(val)) return true;
+  if (['false', 'f', '0', 'n', 'no'].includes(val)) return false;
+  return null;
+}
+
+function validateOptionalBoolean(value?: string) {
+  const bool = parseBoolean(value);
+  if (bool === null) {
+    return 'Please enter a boolean value (e.g. yes, no, true, false) or leave it empty';
+  }
+  return true;
```

Comment:
Shouldn't it be:

```suggestion
  return bool;
```

Reply:
yeah... if you return a string it will be an error message that will be printed to the user. If the value is valid you just return `true` 🤷